packages/components/pie-icon-button/package.json

```diff
@@ -30,7 +30,7 @@
     "@justeattakeaway/pie-webc-core": "workspace:*"
   },
   "peerDependencies": {
-    "@justeat/pie-design-tokens": "5.5.0"
+    "@justeat/pie-design-tokens": "^5.5.0"
```

Comment:
Did you mean to add the caret on this one?

Reply:
Yes. At the moment `pie-icon-button` seems to be the only component that has a peerDependency on the design tokens. I wanted to use the latest version of the tokens in storybook but yarn was complaining that the peer dependency didn't match, so I thought the caret would mean we don't have to come in here and update it all the time, unless we publish a new major version.